src/OpenTelemetry.Instrumentation.Process/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 0.5.0-beta.1
+
+Released 2023-Feb-17
```

Comment:
Please make a note that old packages ware delisted in nuget org

Reply:
@Yun-Ting could you make a note here? and then I'll push the new nuget and delist old.

Reply:
updated the changelog; thank you